h/groups/views.py

```diff
@@ -15,13 +15,15 @@
 
 @view_config(route_name='group_create',
              request_method='GET',
-             renderer='h:groups/templates/create.html.jinja2',
-             permission="authenticated")
+             renderer='h:groups/templates/create.html.jinja2')
 def create_form(request):
     """Render the form for creating a new group."""
     if not request.feature('groups'):
         raise exc.HTTPNotFound()
 
+    if request.authenticated_userid is None:
+        raise exc.HTTPNotFound()
```

Comment:
It's a fairly common approach that we've been taking on the HTML representations all along. It keeps information about whether something exists or not private.